src/config/default.toml

```diff
@@ -0,0 +1,3 @@
+[meta]
```

Comment:
What does this file for? 👀

Reply:
I see. It's for risedev. Is this the same as the default configuration if no config file is provided? If nope, then it sounds ambiguous. :(

Reply:
> Do we ensure that this is the same as the default configuration if no config file is provided?

No 🥶 Can you help it?